### PR TITLE
v1.5.2 Release

### PR DESCRIPTION
**Pull request by RBirmiwal**
- Started documentation updates
- MFKans
- Load forecasting

**Before merge I will remove README_old.md, that is just there incase we want to revert documentation this release**

### CODE REVIEW

**Review by drgona — APPROVED** [review]
Well done everyone! @brunopjacob @RBirmiwal @caryfaulknerPNNL @nicoleeman 
There are only minor comments to improve the accessibility of our examples by including more markup cells especially when introducing new methods and architectures. But this is something we could improve in most of our examples.

File: examples/KANs/p1_fbkan_vs_kan_noise_data_1d.ipynb
[the remark this answers is not in the document]
Comment (brunopjacob) [reply examples/KANs/p1_fbkan_vs_kan_noise_data_1d.ipynb:1]: I've added a bit more details there. Please let me know your thoughts!

File: examples/KANs/p2_fbkan_vs_kan_noise_data_2d.ipynb
[the remark this answers is not in the document]
Comment (brunopjacob) [reply examples/KANs/p2_fbkan_vs_kan_noise_data_2d.ipynb:1]: Same thing, I've added details. Please take a look and let me know if that's more aligned with what you were thinking! Thanks!

[the remark this answers is not in the document]
Comment (drgona) [reply examples/KANs/p2_fbkan_vs_kan_noise_data_2d.ipynb:1]: Thank you Bruno. I think we are ready to deploy this release.

File: examples/PDEs/Part_6_PINN_NavierStokesCavitySteady_KAN.ipynb
[the remark this answers is not in the document]
Comment (brunopjacob) [reply examples/PDEs/Part_6_PINN_NavierStokesCavitySteady_KAN.ipynb:1]: I've added some descriptions of the results.

**Review by brunopjacob — APPROVED** [review]

**Comment by drgona** [comment]
This research was partially supported by the Energy Efficiency and Renewable Energy, Building Technologies Office under the “Dynamic decarbonization through autonomous physics-centric deep learning and optimization of building operations” and the “Advancing Market-Ready Building Energy Management by Cost-Effective Differentiable Predictive Control” projects. This project was also supported from the U.S. Department of Energy, Advanced Scientific Computing Research program, under the Uncertainty Quantification for Multifidelity Operator Learning (MOLUcQ) project (Project No. 81739).
PNNL is a multi-program national laboratory operated for the U.S. Department of Energy (DOE) by Battelle Memorial Institute under Contract No. DE-AC05-76RL0-1830.

**Pull request merged**